Sources/libcoverage/coverage.c

```diff
@@ -19,11 +19,17 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#if !defined(_WIN32)
```

Comment:
Nit: I think I would slightly prefer to have fewer `#if`s here if possible, even if that breaks the sorting of the includes. Maybe we can just have a 2nd block of includes inside a single `#if !defined(_WIN32)` after line 33, would that work?
If not, I'm also fine leaving it as it is though!

Reply:
SGTM, updated.